### PR TITLE
Add support of 'format' parameter to breakpoints transformation

### PR DESCRIPTION
**Pull request by RTLcoil**

### CODE REVIEW

File: cloudinary/utils.py [hunk 1]
```diff
@@ -284,6 +284,10 @@ def breakpoint_settings_mapper(breakpoint_settings):
     transformation = breakpoint_settings.get("transformation")
     if transformation is not None:
         breakpoint_settings["transformation"], _ = generate_transformation_string(**transformation)
+        if breakpoint_settings.get("format"):
```
Comment (tocker) [inline cloudinary/utils.py]: An empty string in `format` is also valid.

Comment (tocker) [reply cloudinary/utils.py]: Best to call `build_eager` and fix the condition there.

File: tests/uploader_test.py [hunk 2]
```diff
@@ -289,13 +289,14 @@ def test_responsive_breakpoints(self):
             responsive_breakpoints=[dict(create_derived=False,
                                          transformation=dict(angle=90)),
                                     dict(create_derived=False,
-                                         transformation=dict(angle=45))])
+                                         transformation=dict(angle=45),
+                                         format="jpg")])
```
Comment (tocker) [inline tests/uploader_test.py]: Add test for empty string in `format`

Comment (RTLcoil) [reply tests/uploader_test.py]: Please review https://github.com/cloudinary/pycloudinary/pull/125/commits/e8aec4fb5d977be45cc79dd147399ce0bc9ac478

**Comment by tocker** [comment]
This change is not relevant. Format is now supported as a separate parameter.
